Jenkinsfile

```diff
@@ -38,6 +38,10 @@ node {
       govuk.rubyLinter("app lib helpers spec bin")
     }
 
+    stage("Lint documentation") {
+      sh "find . -name '*.md' |xargs ./vale-lint.sh"
```

Comment:
Vale has a `--glob` option. Could we use that instead?

```sh
vale --glob='*.md' .
```

If we do, it avoids using the separate shell script, and avoids the cycle of "fix file → check → fail on new file → fix new file..." that happens at the moment, because the script exits on the first failure.

Reply:
Ah, unless you did it like that on purpose to only highlight errors, rather than outputting a deluge of warnings on every run? Might want to tweak the shell script to output all failures then, rather than exiting on the first one it finds.

Reply:
I did it like that I think originally for use with pre-commit hooks, but CI it's different so glob may be more appropriate as like you said it does fail -> fix -> fail -> fix. 

I'll see if I can suppress all the warnings as well so we just throw errors.